libV2/server/middleware.js

```diff
@@ -70,15 +81,7 @@ function errorMiddleware(err, req, res, next) {
             message,
         },
     });
-}
-
-function responseLoggerMiddleware(req, res, next) {
-    const info = {
-        httpCode: res.statusCode,
-        httpMessage: res.statusMessage,
-    };
-    req.logger.end('finished handling request', info);
-    return next();
+    responseLoggerMiddleware(req, res);
```

Comment:
should this pass in next?

Reply:
We stop calling `next` in `errorMiddleware` because we write out all our responses there and want to halt the chain. I chose to make the `next` param optional for `responseLoggerMiddleware` rather than passing a no-op in.